0x48piraj/dist/loading.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
purpose of this file?

Reply:
While our documents get ready, we can embed this design until then.